Examples/Physics_applications/plasma_acceleration/README.rst

```diff
@@ -46,17 +66,29 @@ For `MPI-parallel <https://www.mpi-forum.org>`__ runs, prefix these lines with `
          :language: ini
          :caption: You can copy this file from ``Examples/Physics_applications/plasma_acceleration/inputs_3d_boost``.
 
-Analyze
--------
+Visualize
+---------
 
-.. note::
+The plasma has a density profile that consists of a long up-ramp followed by a uniform plateau.
 
-   This section is TODO.
+With this configuration the driver excites a nonlinear plasma wake and drives the bubble depleted of plasma electrons where the beam accelerates, as can be seen in Fig
+.. figure:: https://user-images.githubusercontent.com/10621396/290962794-7bdbbff7-39fa-4d2e-8b5f-2030f811d257.png
 
```

Comment:
I'm not 100% sure, but I think the empty line between `.. figure::` and `:alt:` will cause some issues.

Unrelated, you could add `:width: 95%` to control how big the figure appears on the page.

Reply:
fixed and done, the energy plot takes a more reasonable amount of page real estate now